_integration-schemas/klaviyo/global_exclusions.md

```diff
@@ -0,0 +1,32 @@
+---
+tap: "klaviyo"
+version: "1.0"
+
+name: "global_exclusions"
+doc-link: ""
+singer-schema: "https://github.com/singer-io/tap-klaviyo/blob/master/tap_klaviyo/schemas/global_exclusions.json"
+description: ""
+
+replication-method: "Full Table"
+
+api-method:
+    name: ""
```

Comment:
This follows the same logic as the comment I left about `api-method` for the `lists` table. To get this data, we're actually using [this endpoint](https://www.klaviyo.com/docs/api/lists#exclusions-list): `https://a.klaviyo.com/api/v1/people/exclusions`